### ISSUE TITLE
dump of public table data to Rodney

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#351]
Per email, get a dump of table data to Rodney.

**Comment by ctm** [comment]
I've fired up a pg_dump of `public_table_messages`.

I should be able to load it into a local database. I'll do the same with `players` and `events` and `tables`, then I'll just use some SQL to delete all rows except the BARGE rows, then do a complete dump of the db and send that to Rodney.

Easy peasy, I think.

**Comment by ctm** [comment]
I haven't started playing with it (I want to exercise soon), but I have the raw data locally):
```
bash-3.2[master]$ ls -l ~/for_rodney/
total 65792
-rw-r--r--  1 ctm  staff     22446 Aug  4 05:41 events.psql.gz
-rw-r--r--  1 ctm  staff     30519 Aug  4 05:41 players.psql.gz
-rw-r--r--  1 ctm  staff  32791309 Aug  4 05:35 public_table_messages.psql.gz
-rw-r--r--  1 ctm  staff      7657 Aug  4 05:42 tables.psql.gz
```

**Comment by ctm** [comment]
It turns out having the files separate made things more difficult for me, so I wound up using:
```
time pg_dump -U mb2 -Z 9 -f ~/for_rodney/tables.psql.gz --no-sync -E utf-8 -t players -t events -t tables -t public_table_messages
bash-3.2$ createdb for_rodney
bash-3.2$ gzcat tables.psql.gz | psql for_rodney
bash-3.2$ psql for_rodney
for_rodney=# delete from public_table_messages where table_id in (select id from tables where event_id not in (select id from events where barge = true));
delete from public_table_messages where table_id in (select id from tables where event_id not in (select id from events where barge = true));
DELETE 1154008
for_rodney=# delete from tables where event_id not in (select id from events where barge = true);
delete from tables where event_id not in (select id from events where barge = true);
DELETE 657
for_rodney=# delete from events where barge != true;
delete from events where barge != true;
DELETE 681
bash-3.2$ pg_dumpall -l for_rodney > all.psql
pg_dumpall -l for_rodney > all.psql
bash-3.2$ ls -l all.psql 
ls -l all.psql 
-rw-r--r--  1 ctm  staff  109043236 Aug  4 06:14 all.psql
bash-3.2$ gzip -9 all.psql 
gzip -9 all.psql 
bash-3.2$ ls -l all.psql.gz 
ls -l all.psql.gz 
-rw-r--r--  1 ctm  staff  8476762 Aug  4 06:14 all.psql.gz
```

I've send Rodney all.psql.gz for him to smoke-test. I'll leave this issue open until I hear back from him.

**Comment by ctm** [comment]
Oops. I should have used `pg_dump` rather than `pg_dumpall`. It's been a while since I've used `pg_dump` and in the old days I used `pg_dumpall` regularly whenever we upgraded our version of Postgresql. I forgot that it dumps all the databases ...

I've asked Rodney to delete what I sent him and I've sent him a new dump that just has the four tables (`players`, `events`, `tables` and `public_table_messages`) I had agreed to send him.
